src/components/taskCreator/RequirementsSelection.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useI18n } from "vue-i18n";
+
+const { t } = useI18n();
+
+const props = defineProps({
+    modelValue: {
+        type: Array as () => string[],
+        required: true
+    },
+});
+
+const emit = defineEmits(["update:modelValue"]);
+
+const requirements = computed(() => [
+    { text: t("taskCreator.requirePhoto"), value: "photo" },
+    { text: t("taskCreator.requireDescription"), value: "description" },
+    { text: t("taskCreator.requireVideo"), value: "video" },
+]);
+
+const updateValue = (value: string[]) => {
+    emit("update:modelValue", value);
+};
+</script>
+
+<template>
+    <v-select :label="$t('taskCreator.requirementsTitle')" :items="requirements" v-model="props.modelValue"
+        @update:model-value="updateValue" multiple item-title="text" item-value="value"
+        :rules="[(v: string[]) => v.length > 0 || $t('taskCreator.requirementsValidation')]"></v-select>
```

Comment:
```suggestion
        :rules="[(v: string[]) => v.length > 0 || $t('taskCreator.requirementsValidation')]" />
```
Nie zostawiajmy pustych klamr